core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerBuilder.java

```diff
@@ -71,8 +71,9 @@
     /**
      * Creates a new {@link CircuitBreakerBuilder} with the specified name.
      *
-     * @param name The name of the circuit breaker.
+     * @deprecated Use {@link CircuitBreaker#builder()} instead.
```

Comment:
- `builder()` -> `builder(String)`?
- `instead` can be omitted - Please fix this for all changes in this PR.

Reply:
Ok, I will remove all "instead."